src/Kunstmaan/MediaBundle/Controller/MediaController.php

```diff
@@ -438,7 +436,6 @@ private function createAndRedirect(Request $request, $folderId, $type, $redirect
      *
      * @Route("create/modal/{folderId}/{type}", requirements={"folderId" = "\d+", "type" = ".+"}, name="KunstmaanMediaBundle_media_modal_create")
      * @Method({"GET", "POST"})
-     * @Template()
      *
```

Comment:
method 'createAndRedirect' can still return an array. In that case it could break. So either 'createAndRedirect' should be adapted or a default template should be provided.

Reply:
Good catch! I'm not sure this method is actually used anymore. I can only find 1 usage in the code, but I don't know yet how to "trigger" that usage. Currently there is no corresponding twig template for this action, so I will look into this.

Reply:
This action was not used as a "GET" anywhere only as a POST for the add file/image/video popup. So I've removed GET as allowed method. Also this code couldn't have worked before as the is no template `createModal.html.twig` or `create_modal.html.twig`

Reply:
Thanks for fixing a [three year old bug](https://github.com/Kunstmaan/KunstmaanBundlesCMS/issues/580)